src/dsp_tools/models/exceptions.py

```diff
@@ -78,7 +128,9 @@ class UserError(BaseError):
 
 
 class XmlError(Exception):
```

Comment:
maybe `XmlUploadError` would be clearer? Like this, it made me think it refers to an error in the XML file.